content/_index.html

```diff
@@ -48,12 +48,21 @@ <h1>Participating</h1>
 
     <p>Looking forward to seeing you!</p>
 
-    <h2>Upcoming meetings</h2>
+    <h2>Upcoming posts</h2>
```

Comment:
I would prefer to keep "meetings" as it indicates to the reader that there is an actual gathering. As a side note, for future meetings, we could also add the location and Zoom link (I will make a PR to update the "Kickoff meeting"). 
Once the meeting is passed and we write the minute of it, it is then placed into the "post" category.

Reply:
It depends on how we want to structure the page. I thought we might put other posts up, too. Like referencing meetings from ETHZ, updates on the kickoff preparation, perhaps other interesting articles.

What do you think?

Reply:
Ok, that makes sense to me !